packages/code-du-travail-api/src/server/routes/search/getEsReferences.js

```diff
@@ -63,6 +63,7 @@ const getEsReferences = async (refs = []) => {
         delete esReference._source.isPublished;
         return esReference;
       }
+      return [];
```

Comment:
adding it, it will remove document that are not found maybe we should discuss what to do rather  before removing them ?

Reply:
100% open to discussion !
Pros: If we find no ref, it "should" mean that that the builded external link will lead to a 404. This is what happens in the api tests on ref on unexisting content. I case thiis happens, I think the user experience is better if we do not display them.
Cons: We don't see on the frontend the broken ref so we might miss them. But isn't it what we want for the end-user ?

Reply:
decision to hide unresolved